### PR TITLE
Suggestions for libraries to add

### PR DESCRIPTION
**Pull request by redoxcode**
Hello everyone,
I want to suggest some of the libraries I created for this list.
These where all created to help using micropython in robotics.
I think they have good documentation and are all downloadable from pypi (so they can used directly from thonny).

- [micropython-ppm_reader](https://github.com/redoxcode/micropython-ppm_reader): No comparable library in the list yet.
- [micropython-servo](https://github.com/redoxcode/micropython-servo): No comparable library in the list yet.
- [micropython-stepper](https://github.com/redoxcode/micropython-stepper): There are some other stepper related libraries in the list. But most lack any documentation :(
- [micropython-dfplayer](https://github.com/redoxcode/micropython-dfplayer). There is an other library with the exact same name in the list (Yikes!). But I think this one is better documented and more importantly it's the one you get when you search for micropython-dfplayer on pypi. So I think it should be included.

Let me know what you think. Thank you!

### CODE REVIEW

**Comment by redoxcode** [comment]
Sorry for adding multiple libraries in one PR. If that is a problem, I can close this one and create 4 individual PRs.

File: readme.md [hunk 1]
```diff
@@ -75,6 +75,7 @@ of required frequency response to a filter implementation.
 * [KT403A-MP3](https://github.com/jczic/KT403A-MP3) - Driver for KT403A, used by DFPlayer Mini and Grove MP3 v2.0.
 * [micropython-buzzer](https://github.com/fruch/micropython-buzzer) - Play Nokia compose and mid files on buzzers.
 * [micropython-dfplayer](https://github.com/ShrimpingIt/micropython-dfplayer) - Driver for DFPlayer Mini using UART.
+* [micropython-dfplayer](https://github.com/redoxcode/micropython-dfplayer) - Library to control the DFPlayer mini mp3 player module.
```
Comment (andypiper) [inline readme.md:77]: mp3 -> MP3 to match common usage and usage elsewhere in the list.

File: readme.md [hunk 2]
```diff
@@ -286,6 +287,10 @@ of required frequency response to a filter implementation.
 * [micropython-ds1302-rtc](https://github.com/omarbenhamid/micropython-ds1302-rtc) - DS1302 RTC Clock driver for MicroPython.
 * [DS3231micro](https://github.com/notUnique/DS3231micro) - MicroPython library for DS3231.
 
+#### RC receiver
+
+* [micropython-ppm_reader](https://github.com/redoxcode/micropython-ppm_reader) - Library to decode PPM signals coming from a RC receiver
```
Comment (andypiper) [inline readme.md]: Line should end with a period (.)

File: readme.md [hunk 3]
```diff
@@ -580,10 +585,12 @@ input via push buttons or a navigation joystick and an optional rotary encoder.
 
 #### Servo
 
+* [micropython-servo](https://github.com/redoxcode/micropython-servo) - Library to control rc servos using direct PWM output in a tidy way.
```
Comment (andypiper) [inline readme.md]: rc -> RC for consistency

**Comment by andypiper** [comment]
Added some proofreading comments. In general the preferred and [requested](https://github.com/mcauser/awesome-micropython/blob/master/contributing.md) PR format is for a single addition per PR, so you may want to rework.

**Comment by redoxcode** [comment]
As suggested I reworked the entries and crated 4 individual PRs.
Thank you!

**Comment by mcauser** [comment]
Thanks @andypiper and @redoxcode

**Pull request closed without merging**